Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/Assets/Labels.txt

```diff
@@ -0,0 +1,1000 @@
+0,tench, Tinca tinca
```

Comment:
should we put this file in a shared folder? it's the same as [this one](https://github.com/microsoft/Windows-Machine-Learning/blob/master/Samples%2FCustomOperator%2Fdesktop%2Fcpp%2FLabels.txt). I think there's a few other copies in the repo as well.

Reply:
Are all the labels.txt the same? We should if so

Reply:
I *think* they are. we should be able to write a quick script to compare.

Reply:
I removed the image and squeezenet model, would like to remove this label file in a separate PR as this touches several other projects, figured should probably do it separately as well